### PR TITLE
add stuff

### PR DESCRIPTION
**Pull request by Ryan-Nelson**

### CODE REVIEW

File: two.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
     <body>
         <article>
-            <header></header>
+            <header>ryan nelson</header>
```
Comment (stevebrownlee) [inline two.html]: Your name should be in an H1 tag

File: two.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
             <section></section>
```
Comment (stevebrownlee) [inline two.html:12]: You did not provide a fact about yourself

File: two.html [hunk 3]
```diff
@@ -8,9 +8,9 @@
...
-            <footer></footer>
+            <footer>cohort 30</footer>
```
Comment (stevebrownlee) [inline two.html]: Missing "student of"

**Review by stevebrownlee — CHANGES_REQUESTED** [review]

File: two.html (hunk 1 above)
Comment (Ryan-Nelson) [reply two.html]: 👍

**Comment by stevebrownlee** [comment]
:shipit:

**Pull request merged**